paper-ripple-behavior.html

```diff
@@ -79,9 +79,11 @@
         }
         var domContainer = rippleContainer === this.shadyRoot ? this :
           rippleContainer;
-        if (opt_triggeringEvent &&
-            domContainer.contains(opt_triggeringEvent.target)) {
-          this._ripple.uiDownAction(opt_triggeringEvent);
+        if (opt_triggeringEvent) {
+          var target = opt_triggeringEvent.target;
+          if (domContainer.contains(/** @type {Node} */(target))) {
```

Comment:
I think what @samccone means is:

```js
/** @type {Node} */
var target = opt_triggeringEvent.target;
```
or
```js
var target = /** @type {Node} */(opt_triggeringEvent.target);
```
I basically didn't do that because I think it's better to leave it as an `EventTarget` where possible, as this is it's true type (i.e. it could be `window` or other non-`Node`s).  The subtle difference just doesn't matter to `contains()`, as far as I know.

We could also do something like:

```js
if (opt_triggeringEvent && opt_triggeringEvent.target instanceof Node)
```

but that doesn't work across frames and incurs a run-time cost.